config/bunny.yml.example

```diff
@@ -21,7 +21,7 @@ default: &default
       subjects:
         volume_tracking:
           subject: 'create-aliquot-in-mlwh'
-          version: 3
+          version: 2
```

Comment:
Just to clarify, why did the Red Panda avro schema version change?

Reply:
Because there is no version 3. I spoke to Dasun and he said he probably left it at 3 while he was testing.